packages/quantic/cypress/fixtures/folded-result-list/exampleResult.json

```diff
@@ -49,7 +49,8 @@
         "foldingchild": ["leaf2"],
         "foldingparent": "node1",
         "urihash": "#1"
-      }
+      },
+      "totalNumberOfChildResults": 2
```

Comment:
Instead of always being true. moreResultsAvailable now depends on the `totalNumberOfChildResults` 

<img width="809" alt="image" src="https://github.com/user-attachments/assets/eaeb569a-fc77-483e-8b66-dc73bc2d8a4a">